dev/com.ibm.ws.ui.tool.serverConfig/resources/WEB-CONTENT/js/components/core.js

```diff
@@ -795,63 +702,6 @@ var core = (function() {
     };
 
 
-    var isServerRunning = function(hostAndServerId) {
```

Comment:
Why did we remove these methods?  (isServerRunning, isStandaloneServerRunning and isCollectiveServerRunning) Without them, there are variables that are no longer defined.